documentatie/use-cases/gsb-genereren-bestanden.md

```diff
@@ -13,13 +13,14 @@ __Precondities:__
 __Hoofdscenario:__
 
 1. De coördinator voert de datum en locatie van de zitting in.
-2. De coördinator voert de aanwezige leden in.
-3. De coördinator voert bezwaren van de keizers in.
-4. De coördinator voert onregelmatigheden of bijzonderheden in.
+2. De coördinator voert de aanwezige leden van het GSB in.
+3. De coördinator voert de bezwaren van kiezers tijdens de zitting in.
+4. De coördinator voert onregelmatigheden of bijzonderheden tijdens de zitting in.
 5. De coördinator voert extra onderzoeken van het GSB in.
 6. De coördinator genereert het concept-PV en het digitaal bestand.
 
 #### Open punten
 
 - Wat zit er precies in het digitaal bestand?
-- Moet de applicatie een preview van het gegenereerde PV tonen, zodat de coördinator die kan controleren?
\ No newline at end of file
+- Moet de applicatie een preview van het gegenereerde PV tonen, zodat de coördinator die kan controleren?
```

Comment:
```suggestion
- Moet de applicatie een preview van het te genereren PV tonen, zodat de coördinator die kan controleren en eventuele fouten kan herstellen?
```
use case is meer: moet de applicatie een preview van het PV tonen, zodat de coördinator eventuele fouten nog kan corrigeren. (controleren is een middel, niet het doel)